PT-7265/Mark JobStatus.CAPTURED as deprecated (#828)
up42/up42-py: CHANGELOG.md

For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).

### 3.6.0a1
**May 20, 2026**
### Deprecated

> - `JobStatus.CAPTURED` is deprecated and will be removed in version `4.0.0`.

Reviewer: ```suggestion
- Deprecate `JobStatus.CAPTURED`. It will be removed in version `4.0.0`.
```
